### PR TITLE
Support for delete key

### PR DESCRIPTION
**Pull request by DKMiller71**
Added support for delete to clear current cell contents

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### Version: Exolve v1.55.1: November 14, 2023
```
Comment (viresh-ratnakar) [inline CHANGELOG.md:3]: s/Version/Minor Version/

File: exolve-m.js [hunk 2]
```diff
@@ -349,6 +349,8 @@ function Exolve(puzzleSpec,
              <b>Ctrl-B:</b> Print crossword, <b>Ctrl-/:</b> Jump to/back-from
              notes, <b>Ctrl-*:</b> Mark clue as fave in notes, adding a *
              prefix.</li>
+         <li><b>Delete:</b>
+						 Clear the content sof the current square.</li>
```
Comment (viresh-ratnakar) [inline exolve-m.js:353]: Delete the extraneous leading space. Typo: s/content sof/contents of/

File: exolve-m.js [hunk 3]
```diff
@@ -5641,6 +5643,14 @@ Exolve.prototype.handleKeyUpInner = function(key, shift=false) {
     this.retreatCursorInLight();
     return true
   }
+  if (key == 46) { // delete key
+    let gridCell = this.currCell()
```
Comment (viresh-ratnakar) [inline exolve-m.js:5647]: s/let/const/

File: exolve.html [hunk 4]
```diff
@@ -396,6 +396,8 @@
              <b>Ctrl-B:</b> Print crossword, <b>Ctrl-/:</b> Jump to/back-from
              notes, <b>Ctrl-*:</b> Mark clue as fave in notes, adding a *
              prefix.</li>
+         <li><b>Delete:</b>
+						 Clear the content sof the current square.</li>
```
Comment (viresh-ratnakar) [inline exolve.html:400]: just re-copy the updated exolve-m.js into this file after fixing the minor typos listed above.

File: exolve-m.js (hunk 3 above)
Comment (DKMiller71) [reply exolve-m.js:5647]: updated

File: exolve-m.js (hunk 2 above)
Comment (DKMiller71) [reply exolve-m.js:353]: updated

File: CHANGELOG.md (hunk 1 above)
Comment (DKMiller71) [reply CHANGELOG.md:3]: updated

File: exolve.html (hunk 4 above)
Comment (DKMiller71) [reply exolve.html:400]: updated

**Comment by viresh-ratnakar** [comment]
Thanks!

**Pull request merged**